test/scripts/cases/_olla.sh

```
@@ -0,0 +1,311 @@
+#!/bin/bash
+# _olla.sh
+# Olla-specific helper functions for test scripts
+
+# Global variable for Olla PID
+OLLA_PID=""
+
+# Function to get git version info for binary naming
+get_git_version() {
+    local current_dir=$(pwd)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)